packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -838,3 +838,13 @@ function endsWith(string, suffix) {
     return string.endsWith(suffix);
   }
 }
+
+Ember.defineProperty(RESTAdapter, 'maxUrlLength', {
```

Comment:
Also wouldn't this break if someone subclassed the adapter to provide their own limit as they would override this getter? cc @rwjblue whats the proper way of doing this